primitives/sidechain/src/validateer.rs

```diff
@@ -48,14 +48,16 @@ impl<OnchainStorage: GetOnchainStorage> ValidateerFetch for OnchainStorage {
 	}
 }
 
-#[cfg(feature = "test")]
-pub mod tests {
+#[cfg(test)]
+mod tests {
 	use super::*;
-	use crate::Header;
 	use codec::Encode;
+	use sp_runtime::{generic::Header as HeaderG, traits::BlakeTwo256};
 	use std::string::ToString;
 	use test_utils::mock::onchain_mock::{validateer_set, OnchainMock};
 
+	pub type Header = HeaderG<u64, BlakeTwo256>;
+
 	pub fn default_header() -> Header {
 		Header::new(
 			Default::default(),
```

Comment:
So this crate can now be tested by `cargo test`? 🥳

Reply:
Yes, we can! 🚀